assets/javascripts/modules/responsiveInlineDetails.js

```diff
@@ -0,0 +1,30 @@
+module.exports = function() {
+
+  $('[data-responsive-inline-details]').each(function() {
+    var $detailsHolder,
+    $detailsTargets,
+    $detailsButton;
+
+    $detailsHolder = $(this);
+    $detailsButton = $detailsHolder.find('.inline-details-toggle');
+    $detailsTargets = $('.' + $detailsHolder.data('responsive-inline-details'));
+
+    $detailsButton.on('click', function(e) {
```

Comment:
Is there a reason for why detailsButtonClick wasn't passed as an argument to $detailsButton.on?
This could be done in a much simpler form
```
$detailsButton.on('click', function(e) {
    e.preventDefault();
    var isCursorActive = $detailsButton.css('cursor') === 'pointer';
    if (!isCursorActive) {
        return;
    }
    $detailsHolder.toggleClass('hide-inline-details-targets');
}
```

or if you want to keep the function separate from the event binding: `$detailsButton.on('click', detailsButtonClick)`